web-common/src/features/dashboards/dimension-table/DimensionHeader.svelte

```diff
@@ -158,5 +213,22 @@
     {#if $exports}
       <ExportDimensionTableDataButton includeScheduledReport={$adminServer} />
     {/if}
+    <Button
+      compact
+      type="text"
```

Comment:
This looks a little funky next to the other buttons in this header. Can you make them consistent? (Either making the others `text` or using the others' style here)